### PR TITLE
[WFLY-16883] Modify description in batch-jberet of subsystem to its c…

### PR DESCRIPTION
**Pull request by moulalis**
Issue: https://issues.redhat.com/browse/WFLY-16883

### CODE REVIEW

File: batch-jberet/src/main/resources/org/wildfly/extension/batch/jberet/LocalDescriptions.properties [hunk 1]
```diff
@@ -50,7 +50,7 @@ batch.jberet.jdbc-job-repository.execution-records-limit=The maximum number of j
   be returned by the job repository.
 
 # Thread pool
-batch.jberet.thread-pool=The thread pool used for batch jobs. Note that the max-thread attribute should always be greater \
+batch.jberet.thread-pool=The thread pool is used for batch jobs. Note that the max-thread attribute should always be greater \
```
Comment (chengfang) [inline batch-jberet/src/main/resources/org/wildfly/extension/batch/jberet/LocalDescriptions.properties:53]: I think the original sentence is fine. It means: this is the thread pool used for batch jobs.

Comment (bstansberry) [reply batch-jberet/src/main/resources/org/wildfly/extension/batch/jberet/LocalDescriptions.properties:53]: @moulalis Can you correct this please? I'd like to get this merged. Thanks.

Comment (moulalis) [reply batch-jberet/src/main/resources/org/wildfly/extension/batch/jberet/LocalDescriptions.properties]: > @moulalis Can you correct this please? I'd like to get this merged. Thanks.

@bstansberry Done. Thanks.

**Review by chengfang — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @moulalis

**Pull request merged**